packages/editor-ui/src/api/users.ts

```diff
@@ -0,0 +1,71 @@
+import { IPersonalizationSurveyAnswers, IRestApiContext, IUserResponse } from '@/Interface';
+import { IDataObject } from 'n8n-workflow';
+import { makeRestApiRequest } from './helpers';
+
+export async function loginCurrentUser(context: IRestApiContext): Promise<IUserResponse | null> {
+	return await makeRestApiRequest(context, 'GET', '/login');
+}
```

Comment:
same as in settings api handler: can't we omit `async/await` and return `makeRestApiRequest`?